.github/workflows/python.yml

```
@@ -2,18 +2,23 @@ name: CI (Python)
 
 on:
   pull_request:
```

```suggestion
  pull_request:
```
I think this line needs to go away to not trigger this job.

---

it's still supposed to trigger it because of the linting that we always want to do. But then there is a check later down  and I just learned how that was broken :)
Or rather I believe I know now why